src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  selectedTodo: Todo | undefined,
```

Comment:
Don't render this component if there is no selectedTodo at all
```suggestion
  selectedTodo: Todo,
```